compass.yaml.example

```diff
@@ -17,14 +17,16 @@ db:
     host: localhost
     port: 5432
     name: compass
-    user: postgres
-    password: ~
+    user: compass
+    password: compass_password
     sslmode: disable
 
 service:
     host: localhost
     port: 8080
+    baseurl: localhost:8080
```

Comment:
wouldn't this make it confusing to have different config of server address for client and server?

Reply:
Yes, moved it to the client when pairing with mas stew.